test/test-pages.js

```diff
@@ -37,7 +37,7 @@ describe('Page status', function () {
 
   describe('User registration page', function() {
     it('status', function(done) {
-      request('http://localhost:3000/registration', function(error, response, body) {
+      request('http://localhost:3000/registration/:badge', function(error, response, body) {
```

Comment:
Don't you need to provide an actual badge here? Since you are making an actual request.

Reply:
Hmmm... that's a good point. It is probably interpreting the badge number as a literal string value, "/:badge", instead of a number, like I meant to specify. The database probably wouldn't like that. I'll update/test it.

Reply:
Also, it looks like there were changes to my package-lock.json. Do you not see them in "Files"? I'm about to rebase and re-submit... so we'll see shortly if this commit includes it. Is it being "ignore"d somewhere maybe?

Reply:
You may need to check the .gitignore, you may need to readd the package lock file somehow

Reply:
Update: @danieleynis the "package-lock.json" file is not in the gitignore file. It does exist (and has quite a lot of stuff in it) on my branch in my local repository - even though it doesn't 'appear' to in the remove repository.

Despite the differences, "git status" said that I was current with my branch in the remote repository (where it 'appears' to be missing data). I did "npm install" to touch the file and force git to recognize it locally, then "git add /package-lock.json". I got the following output:
![image](https://user-images.githubusercontent.com/33047755/35764001-c3ff266c-086c-11e8-92c7-77b8bff0b206.png)

That warning raised my eyebrow so I did some looking online and am wondering if it may just be interpreting the file wrong due to our cross-platform development (Unix and Windows interpret the end of a line differently: a line feed (LF) versus a carriage return and line feed (CRLF), respectively. The documentation suggests our Windows developers should set 
```$ git config --global core.autocrlf true```
...and the Unix and/or Mac developers should use
```$ git config --global core.autocrlf input```

I don't know what else it could be... could it hurt to give that a try?
<https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#Formatting-and-Whitespace>
![image](https://user-images.githubusercontent.com/33047755/35764051-c6cdccc6-086d-11e8-9d4b-0590f095ae00.png)